src/main.c

```diff
@@ -334,6 +350,55 @@ static void setup_adc(void)
 
 
 /**
+ * @brief TIM1 Interruption routine
+ *
+ *- Update event interrupt flag management
+ *- Led toggling each second
+ */
+void tim1_up_isr(void)
+{
+	int j = 0;
+
+	if (timer_get_flag(TIM1, TIM_SR_UIF)) {
+		timer_clear_flag(TIM1, TIM_SR_UIF);
+		gpio_toggle(GPIOA, GPIO7);
+	}
+}
+
+
+/**
+ * @brief Timer 1 (TIM1) setup.
+ *
+ * The timer 1 generates an update event interruption that invokes the
+ * function tim1_up_isr.
+ * - Enable TIM1 interruption due an update event on the system.
```

Comment:
What do you mean? I think the sentence might be incorrect (in English I mean).

Reply:
you are right, I rewrote the sentence to be clear "Enable an interruption of type timer 1 update event on the system"